qualtran/bloqs/arithmetic/comparison.py

```diff
@@ -875,6 +875,25 @@ def wire_symbol(
             return TextBox('t⨁(a>b)')
         raise ValueError(f'Unknown register name {reg.name}')
 
+    def build_call_graph(self, ssa: 'SympySymbolAllocator') -> Set['BloqCountT']:
+        if self.bitsize == 1:
+            return {(MultiControlX(cvs=(1, 0)), 1)}
+        # Ignore Allocation and Deallocation bloqs for now.
```

Comment:
I think we typically ignore these in `build_call_graph`, so not worth noting.

Could you add a quick `qlt_testing.assert_equivalent_bloq_counts` test?

Reply:
done